tests/utils.py

```diff
@@ -8,3 +11,20 @@ class UnmockedRequestException(GeneralTestException):
 
 class UnexpectedMockInvocationException(GeneralTestException):
     pass
+
+
+def create_measure_page_versions(db, example_measure_page, required_versions):
+    for page_version in required_versions:
+        page = Page(
+            guid="test",
+            version=page_version,
+            parent_guid=example_measure_page.parent.guid,
+            parent_version=example_measure_page.parent.version,
+            page_type="measure",
+            uri="test-measure-page-2",
+            title="Test measure page 2",
```

Comment:
The title could maybe be paramaterised to include the `page_version` and then the test above could have a list of unique items rather than `"Test measure page 2", "Test measure page 2"`

Reply:
Ok, done.